From Zero to Robert De Niro/Program.cs

```diff
@@ -66,15 +69,11 @@ static void CheckEven()
                     Console.WriteLine("Це не число. Спробуйте ще раз.");
                 }
             }
+
             if (num % 2 == 0)
-            {
                 Console.WriteLine("Число парне.");
-            }
             else
-            {
```

Comment:
You can use it like this, no syntax issue, but according to microsoft conventions it's better to use braces. https://dotnettips.wordpress.com/2023/06/27/microsoft-net-code-analysis-always-add-braces-in-c/